apisix/admin/utils.lua

```diff
@@ -78,4 +80,28 @@ function _M.fix_count(body, id)
 end
 
 
+function _M.decrypt_params(decrypt_func, body, schema_type)
+    -- list
+    if body.list and #body.list > 0 then
```

Comment:
For the same reason, we can skip the `and #body.list > 0` too.

Reply:
done